### PR TITLE
[solidity/it-it] Solidity italian translation

### PR DESCRIPTION
**Pull request by al-ias**
- [X] I solemnly swear that this is all original content of which I am the original author
- [X] Pull request title is prepended with `[language/lang-code]` (example `[python/fr-fr]` or `[java/en]`)
- [X] Pull request touches only one file (or a set of logically related files with similar changes made)
- [X] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

Notes:
- I left some technical terms in English, because in Italy we tend to use the English expressions (I think the Italian translation would be counter-intuitive)
- I chose not to translate the code in section 9.G (it could be less readable)
- I didn't translate the images and the code linked to the Remix IDE (and Gists). Maybe done in a separate PR.

### CODE REVIEW

File: it-it/solidity.html.markdown [hunk 1]
```diff
@@ -0,0 +1,1069 @@
+---
+language: Solidity
+filename: learnSolidity.sol
```
Comment (mribeirodantas) [inline it-it/solidity.html.markdown:3]:
```suggestion
filename: learnSolidity-it.sol
```

**Review by mribeirodantas — COMMENTED** [review]
You must change the filename, otherwise the translated document/sourcecode would interfere with the original document/sourcecode.

**Comment by al-ias** [comment]
> You must change the filename, otherwise the translated document/sourcecode would interfere with the original document/sourcecode.

Thanks a lot, I didn't notice. I just committed the change your suggested.

**Comment by mribeirodantas** [comment]
Thanks, @al-ias. I'm glad to merge your PR 😄 . Thanks for the contribution!

**Pull request merged**